network-mux/src/Network/Mux/Types.hs

```diff
@@ -255,3 +258,15 @@ muxBearerAsChannel bearer ptclNum ptclDir =
 
       noTimeout :: TimeoutFn m
       noTimeout _ r = Just <$> r
+
+--
+-- Errors
+--
+
+data MuxRuntimeError =
+    ProtocolAlreadyRunning    !MiniProtocolNum !MiniProtocolDir !MiniProtocolStatus
+  | UnknownProtocol           !MiniProtocolNum !MiniProtocolDir
```

Comment:
This might be caused by a frame with unknown protocol, in which case it shouldn't be here.